browser-test/src/admin/northstar_admin_bulk_update_statuses.test.ts

```diff
@@ -67,19 +68,11 @@ test.describe('with program statuses', {tag: ['@northstar']}, () => {
   test('if more than 100 applications, only the first page of applications undergo status update', async ({
     page,
     adminPrograms,
-    applicantQuestions,
     adminProgramMigration,
+    seeding,
   }) => {
-    test.slow()
-    // There is already 1 application from the beforeEach, so apply 105 more times.
-    for (let i = 0; i < 105; i++) {
-      await logout(page)
-
-      // Submit an application as a guest.
-      await applicantQuestions.applyProgram(programName, true)
-      await applicantQuestions.submitFromReviewPage(true)
-    }
-    const id = await adminPrograms.getApplicationId()
+    // There is already 1 application from the beforeEach, so create 105 more using bulk seeding.
+    await seeding.seedApplications(slugify(programName), 105)
```

Comment:
(blocking) Move the slugify call into the `seedApplications` that way we know it'll always be handled.